src/decawave/launch/decawave_node.launch

```diff
@@ -3,14 +3,13 @@
   <!-- decawave_node.launch
        VERSION 0.0.0 Last Changed 2019-04-01
   -->
-  <!-- Interface: 
+  <!-- Interface:
     Pub: odometry/gps (nav_msgs/Odometry)
-      The position estimate of the decawave, with variances. 
+      The position estimate of the decawave, with variances.
     Param: frequency (double) 50.0; The frequency of the read loop
   -->
   <node pkg="decawave" type="decawave_node" name="decawave_node" output="screen" respawn="true">
     <param name="frequency" value="50.0"/>
-    <param name="port_num" value="0"/>
+    <param name="port_name" value="ttyACM0"/>
```

Comment:
You probably need to have an ID (either string or int) other than port name; that's more robust & useful.